media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,153 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+.firefox-friends {
+    #wrapper {
+        padding-bottom: 0;
+        width: 100%;
+    }
+    #masthead {
+        background: #373737 url(/media/img/contribute/friends/header-img-wide.jpg) 50% 0 no-repeat;
+        padding: 0 180px;
+        width: 100%;
+        .border-box();
+
+        @media only screen and (max-width: 1100px) {
+            padding: 0 100px;
```

Comment:
Is it worth making some mobile-sized images for the background, to help avoid small screens needing to load in the full sized image?